#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_uml_switch - Mattia Dongili <malattia@debian.org>
## rediffed from previous mdz patch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use Debian's standard location for uml_net helper

@DPATCH@

# Author: malattia
# Status: none

--- linux-source-2.6.24/arch/um/drivers/daemon_kern.c.old	2008-02-03 15:09:48.497911268 +0900
+++ linux-source-2.6.24/arch/um/drivers/daemon_kern.c	2008-02-03 15:13:05.546916771 +0900
@@ -65,7 +65,7 @@
 
 	*init = ((struct daemon_init)
 		{ .sock_type 		= "unix",
-		  .ctl_sock 		= "/tmp/uml.ctl" });
+ 		  .ctl_sock 		= "/var/run/uml-utilities/uml_switch.ctl" });
 
 	remain = split_if_spec(str, mac_out, &init->sock_type, &init->ctl_sock,
 			       NULL);
